### DISCUSSION TITLE
Challenging to link directly to a map location.

### DISCUSSION THREAD

**Discussion by sodacrackers** [jaybo/deepzoom-blog#32] [General]
The sidebar help text reads "Login to save or share trips via social media or share a link via email."

Still, it's unclear how to link directly to a map centered on a particular location. Is creating a sharable link possible?

**Comment by jaybo** [comment]
A "trip" is the thing which is sharable.  

When you save a trip, it can contain routes, markers, and tracks.  It will also contain the map center, zoom setting and map layers enabled if you check the option **_Include view settings_**.

![image](https://github.com/jaybo/deepzoom-blog/assets/1607323/155420b1-6dda-42c3-bd62-4cbdf6d5e654)


So just create a new trip with no routes or markers, and then save it with **_Include view settings_** checked, and then share the link.

**Comment by jaybo** [comment]
Also, as of version 3.14.xx, larger screens display a link icon at the top left.  This is a simple way to link to a particular location without the rigamarole of saving a trip.

![image](https://github.com/jaybo/deepzoom-blog/assets/1607323/762c5ffc-d475-43e8-aa76-83c8f72191ca)
